src/js/wysiwygEditor.js

```diff
@@ -434,6 +437,14 @@ class WysiwygEditor {
     });
   }
 
+  _togglePlaceholder() {
+    if (this.get$Body()[0].textContent === '') {
```

Comment:
Please change condition. 
```js
if (this.get$Body()[0].textContent)
```
This is our coding convention.

Also do not call `this.get$Body()` every time in this function, but assign it to a variable in this function like this:
```js
const $body = this.get$Body();
if ($body[0].textContent) {
    $body.addClass('tui-editor-contents-placeholder');
} else {
    $body.removeClass('tui-editor-contents-placeholder');
}
```